apps/src/templates/GameButtons.jsx

```diff
@@ -19,12 +21,30 @@ export const FinishButton = () => (
   </button>
 );
 
+const sendAnalyticsEvent = () => {
+  const isSignedOut =
+    document
+      .querySelector('script[data-issignedout="true"]')
+      ?.getAttribute('data-issignedout') === 'true';
+
+  if (isSignedOut) {
+    analyticsReporter.sendEvent(
+      EVENTS.RUN_BUTTON_PRESSED_SIGNED_OUT,
+      {},
+      PLATFORMS.STATSIG
+    );
+  }
+};
+
 export const RunButton = Radium(props => (
   <button
     type="button"
     id="runButton"
     className={classNames(['launch', 'blocklyLaunch', props.hidden && 'hide'])}
     style={props.style}
+    onClick={() => {
```

Comment:
The fact that this is the first onClick event makes the actual functionality of the run button very mysterious

Reply:
We were just talking to Sanchit about this in person- not trying to override the event that is off floating in the cloud!

Reply:
Sounds good! I'd love to hear the tl;dr of that conversation when you can!

Reply:
Per Bethany's suggestion I'm going to move this file/event and the Music Lab file/event into a separate PR. We'll continue investigating there 🔎